app/scripts/filters/resources.js

```diff
@@ -955,6 +955,17 @@ angular.module('openshiftConsole')
       return annotationFilter(build, 'jenkinsBuildURL');
     };
   })
+  .filter('jenkinsInputURL', function(jenkinsBuildURLFilter) {
+    return function(build) {
+      // TODO we should have a jenkinsInputURL annotation, waiting on
+      // https://github.com/fabric8io/openshift-jenkins-sync-plugin/issues/94
+      var buildURL = jenkinsBuildURLFilter(build);
+      if (!buildURL) {
+        return null;
+      }
+      return new URI(buildURL).segment("/input/").toString();
```

Comment:
Agree this is a stopgap, we aren't closing the issue against the sync plugin

On Jul 11, 2016 7:31 PM, "Sam Padgett" <notifications@github.com> wrote:

> In app/scripts/filters/resources.js
> <https://github.com/openshift/origin-web-console/pull/151#discussion_r70355763>
> :
>
> > @@ -955,6 +955,17 @@ angular.module('openshiftConsole')
> >        return annotationFilter(build, 'jenkinsBuildURL');
> >      };
> >    })
> > +  .filter('jenkinsInputURL', function(jenkinsBuildURLFilter) {
> > +    return function(build) {
> > +      // TODO we should have a jenkinsInputURL annotation, waiting on
> > +      // https://github.com/fabric8io/openshift-jenkins-sync-plugin/issues/94
> > +      var buildURL = jenkinsBuildURLFilter(build);
> > +      if (!buildURL) {
> > +        return null;
> > +      }
> > +      return new URI(buildURL).segment("/input/").toString();
>
> I'm hesitant to assume anything about the Jenkins URL patterns since they
> could change. I don't see this as a documented API.
>
>
> https://github.com/jenkinsci/pipeline-stage-view-plugin/tree/master/rest-api
>
> I hope the sync plugin can add this, and this is only a stop-gap.
>
> @bparees <https://github.com/bparees> FYI
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/151/files/29f01278f270637953c38a1e279391ffc9317bed#r70355763>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABZk7cYUhnm9pa1qQO9_qS2a_Z6XG9z7ks5qUtJXgaJpZM4JJzFt>
> .
>